packages/api/src/external/commonwell/link/shared.ts

```diff
@@ -47,48 +46,16 @@ export function patientWithCWData(
   return patientWithCW;
 }
 
-async function shouldDowngradeLink(link: NetworkLink): Promise<boolean> {
+function isInsideOrgExcludeList(link: NetworkLink, orgIdExcludeList: Set<string>): boolean {
   const identifiers = link.patient?.identifier || [];
-  const checkPromises = identifiers.map(async id => {
+  return identifiers.some(id => {
     const idSystem = id.system?.replace(/^urn:oid:/, "");
```

Comment:
nit at this point, but fixed regexes can be stored outside functions to avoid being compiled on each call.